src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/controllers/AppExceptionHandler.java

```diff
@@ -42,4 +49,26 @@ public ErrorResponse<String> mappingToDTOException(MappingToDTOException excepti
 
         return new ErrorResponse<>(exception.getMessage());
     }
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(ValidationException.class)
+    public ErrorResponse<List<ValidationError>> validationException(ValidationException exception) {
+
+        return new ErrorResponse<>(exception.getMessage(), exception.getErrors());
+    }
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ErrorResponse<List<ValidationError>> handleValidationExceptions(MethodArgumentNotValidException exception) {
+        List<ValidationError> errors = new ArrayList<>();
+
+        for (FieldError error : exception.getBindingResult().getFieldErrors()) {
+            errors.add(new ValidationError(
+                    error.getField(),
+                    error.getDefaultMessage(),
+                    error.getRejectedValue()));
+        }
+
+        return new ErrorResponse<>("Result of validate", errors);
```

Comment:
Zmień message na "Validation erros" czy coś takiego - możesz też wstawić przykładową odpowiedź to obczaję :D